Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.10 as builder
+
+WORKDIR ${GOPATH}/src/github.com/dispatchframework/dispatch-events-aws
+
+COPY ["driver.go", "Gopkg.lock", "Gopkg.toml", "./"]
+
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN dep ensure
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o dispatch-events-aws
+RUN cp ./dispatch-events-aws /dispatch-events-aws
+
+
+FROM scratch
+
+ADD cacert-2018-03-07.pem /etc/ssl/certs/
```

Comment:
So the only thing to make certs work in `scratch` image is to copy this file? Sweet!

Reply:
Yeah, I got stuck by this for a while, until I found some blogs mentioned to do this